tools/topology/m4/igo_nr.m4

```diff
@@ -0,0 +1,66 @@
+divert(-1)
```

Comment:
This should go to topology commit.

Reply:
actually both commits have topology files: one is general change and the other is customized change for sof-tgl-max98373-rt5682.m4. `igo_nr.m4` has to be there for arbitrary topology so I put it inside the audio commit.